pkg/event/asgterminate/v1/parser.go

```diff
@@ -27,7 +27,6 @@ import (
 const (
 	source             = "aws.autoscaling"
 	detailType         = "EC2 Instance-terminate Lifecycle Action"
-	version            = "1"
```

Comment:
Are we just no longer depending on version number for anything?

Reply:
Temporarily. Need to look into why the events don't have the expected version number. Later, if it makes sense, we can restore the checks.